### PR TITLE
ARC-1359 Brahmos Wargames integration

### PR DESCRIPTION
**Pull request by yevhshevchenko**
**What's in this PR?**

**Why**

**Added feature flags**

**Affected issues**  
_Jira Issues_

**How has this been tested?**  
_Include how to test if applicable_

**Whats Next?**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,10 @@ RUN apk add g++ make python3
 RUN apk add --update --upgrade busybox libretls openssl zlib curl
 
 COPY . /app
+
+# Add this the service Dockerfile, at the final stage if multi-stage
```
Comment (gxueatlassian) [inline Dockerfile:11]: Hi from this project we can tell this file is not being used during deployment to ddev/stg/prod
https://bitbucket.org/atlassian/github-for-jira-deployment/src/master/

It should the `Dockerfile.prod`

File: github-for-jira.sd.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ description: Connect app for integrating GitHub into Jira
 organization: Engineering-DevOps
 notifications:
   email: fusion-arc@atlassian.com
+tags:
+  brahmos: enabled
```
Comment (gxueatlassian) [inline github-for-jira.sd.yml:8]: /nit
Can you please help me understand, that this tag will not impact the performance on our prod system? Or unless we tell brahmos to run on prod, everything will be as usuall?

**Review by gxueatlassian — COMMENTED** [review]
nice

File: github-for-jira.sd.yml (hunk 2 above)
Comment (joshkay10) [reply github-for-jira.sd.yml:8]: brahmos isn't supported in prod yet so if we could tag it for only stage that would be ideal! 
we have some `environmentOverrides` in there already, but never tried to access that top level values :shrug:

File: Dockerfile (hunk 1 above)
Comment (yevhshevchenko) [reply Dockerfile:11]: @gxueatlassian do we have a separate dockerfile for stg and ddev?

Comment (gxueatlassian) [reply Dockerfile:11]: I think `ddev` `stg` and `prod` shared the same docker file, but you can adjust that in the deployment scripts above.

File: github-for-jira.sd.yml (hunk 2 above)
Comment (yevhshevchenko) [reply github-for-jira.sd.yml:8]: @joshkay10 I think that we cannot apply global props on environmentOverrides

Comment (yevhshevchenko) [reply github-for-jira.sd.yml]: @joshkay10 some updates. It works!

**Review by gxueatlassian — DISMISSED** [review]

File: github-for-jira.sd.yml (hunk 2 above)
Comment (gxueatlassian) [reply github-for-jira.sd.yml:8]: Hi @yevhshevchenko so this is back to global settings now?

Comment (yevhshevchenko) [reply github-for-jira.sd.yml:8]: Hi @gxueatlassian, yeah it was a problem when I tried to execute it and faced with an error(

**Review by bgvozdev — APPROVED** [review]

File: Dockerfile.prod [hunk 3]
```diff
@@ -32,6 +33,8 @@ RUN apk add curl
 
 USER node
 COPY --chown=node:node --from=build /app /app
+# Add this the service Dockerfile, at the final stage if multi-stage
+COPY --from=docker.atl-paas.net/sox/brahmos-deps/stress-ng:latest /usr/bin/stress-ng /usr/bin/stress-ng
```
Comment (bgvozdev) [inline Dockerfile.prod:37]: Was wondering if it is safe to put it to prod Dockerfile though...

Comment (yevhshevchenko) [reply Dockerfile.prod:37]: @bgvozdev as i understood we do not have a separate file for stg image. I think that it shouldn't be a problem, we just will not have a brahmos on a prod instance

**Pull request merged**